docs/advanced-features/codemods.md

```diff
@@ -64,7 +64,97 @@ Dangerously migrates from `next/legacy/image` to the new `next/image` by adding
 - Removes `objectPosition` prop and adds `style`
 - Removes `lazyBoundary` prop
```

Comment:
This section still says (experimental) - should we update that and link in upgrade guide?

Reply:
@leerob This codemod is experimental and will remain that way forever. Because static analysis can't capture every dynamic usage of the component so users should be aware that its experimental. (similar to the existing [`cra-to-next`](https://nextjs.org/docs/advanced-features/codemods#cra-to-next-experimental) codemod)